src/components/settings/TranslatorMode.svelte

```diff
@@ -38,7 +43,8 @@
     {#if $doAutoPrefix}
       <TextField
         bind:value={$autoPrefixTag}
-        label="Tag to prepend ($filterLang is replaced by your filter language)"
+        label="Tag to prepend ($filterLang is replaced by your first filter language)"
+        rules={autoPrefixRules}
       />
```

Comment:
Should have a dropdown to select a specific language tag instead. This currently makes it unclear which language exactly is being used, and a translator might for example want to still see EN translations but send translations in another language.